Core/Core/Configuration/Config.swift

```diff
@@ -20,22 +21,23 @@ public class Config {
         URL(string: "\(baseURL.description)/privacy")
     }()
     
-    public let feedbackEmail = "support@example.com"
+    public let feedbackEmail = "contact@yam-edu.com"
```

Comment:
This line slipped in... I think you meant to exclude this.

Reply:
Thanks-- fixed!